docs/gettingstarted.md

```diff
@@ -122,6 +122,13 @@ To specify only installing certain groups, use the `--with` and `--without` argu
 
     bundle install --with windows --without docker
 
+## Generate and locally view docs ##
+
+To view the docs that were originally generated by this repository and hosted by Salt Project, the following should generate a local copy of the documentation.
+
+    # Generate HTML documentation of kitchen-salt
```

Comment:
This should say something like `To start a local http server to view the rendered docs` since it serves them up with this command, not just generates them.

Reply:
also probably tell them to `rm -rf html/` first as it doesn't automatically detect out of sync docs if you git pull or something like that.